src/current/v25.1/deployment-operations-skills-taxonomy.md

```diff
@@ -0,0 +1,84 @@
+
+## Cluster maintenance
+
+- [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+- [Handling unplanned node outages]({% link {{ page.version.version }}/recommended-production-settings.md %}#load-balancing)
+- [Adding nodes]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster)
+- [Removing nodes]({% link {{ page.version.version }}/node-shutdown.md %}?filters=decommission#remove-nodes)
+- [Add a region]({% link {{ page.version.version }}/alter-database.md %}#add-regions-to-a-database)
+- [Remove a region]({% link {{ page.version.version }}/alter-database.md %}#drop-region)
+- [Rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade)
+- Downgrade a cluster from a [patch version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-patch-upgrade)
+- Downgrade a cluster from a [major version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade)
+- [Change a cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#change-a-cluster-setting)
+- Cluster repaving involves the following individual skills, which are also used during [rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade):
+    1. [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+    1. Detach the [persistent volume]({% link {{ page.version.version }}/kubernetes-overview.md %}#kubernetes-terminology) (a.k.a. persistent disk) from the removed node's virtual machine (VM) (this step is optional but recommended)
+    1. Delete the removed node's VM
+    1. Start a new VM
+    1. Reattach the persistent disk to the new VM (necessary if you did step #2)
+    1. [Add a node to the cluster]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster) from the new VM
+
+## Troubleshooting
+
+- [SQL response time for specific queries]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#query-issues)
+- [SQL throughput degradation across the board]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#low-throughput)
+- [Cluster instability: Dead/suspect nodes]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues)
+- [Out of memory problems]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#out-of-memory-oom-crash)
+- [Imbalanced cluster load]({% link {{ page.version.version }}/architecture/replication-layer.md %}#load-based-replica-rebalancing)
+- [EOF errors]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#client-connection-issues)
+- [Changefeed is falling behind]({% link {{ page.version.version }}/advanced-changefeed-configuration.md %}#lagging-ranges)
+- [Get a "debug zip" file]({% link {{ page.version.version }}/cockroach-debug-zip.md %})
```

Comment:
Is the term "debug zip" commonly jargon for our customers? I'd think we should spell it out more clearly in this context, like "Download an archive for debugging"

Reply:
Unless support is regularly asking customers to download a "debug zip" so we know that's the terminology they're looking for.

Reply:
yes its def the term everyone uses including in convos with customers